src/utils/extractWhiteboard.js

```diff
@@ -0,0 +1,30 @@
+const extractWhiteboardTags = whiteboard => {
+  // find occurence like lang=perl,js or lang=perl/js
+  const re = /(\w+)=(\w+)[,|/](\w+)/;
```

Comment:
I'm thinking we might face whiteboard tag of the form `[lang=js, xul]` (notice the space after the comma). See if it is easy to integrate this edge case. If it's too difficult, then it should be okay.